src/Microsoft.AspNet.Authentication/ClaimsTransformationAuthenticationHandler.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
> I am currently fixing things so they don't even have to think about 'CookieMiddleWare'.
This change moves us away from the simple model and forces the user to know about types and code not associated with what they are trying to do.

Smells oil out there :smile: